gwm_create_service_account.py

```diff
@@ -305,6 +306,13 @@ async def download_service_account_key():
   await retryable_command(command)
 
 
+async def delete_key():
+  input("\nPress any key after you have downloaded the file.")
```

Comment:
`input()` [reads a line](https://docs.python.org/3/library/functions.html#input), so this needs to say "press Enter".

Reply:
Done